third_party/k8s.io/kubernetes/hack/lib/etcd.sh

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/env bash
```

Comment:
Why are we including etcd.sh and these other files? Are they all direct/indirect dependencies?

Reply:
Yes unfortunately per https://github.com/open-policy-agent/gatekeeper/pull/2461#discussion_r1063725495 .